CheckoutSDK/Classes/CheckoutViewController.swift

```diff
@@ -133,7 +133,8 @@ public class CheckoutViewController: UIViewController {
     
     private func parseTransactionResponse(dict: [String : Any]) -> TransactionResponse?  {
         if let data = dict["data"] as? [String : Any] {
-            guard let id = data["trans"] as? String, let reference = data["reference"] as? String else { return nil}
+            guard let reference = data["reference"] as? String else {return nil}
+            let id = data["trans"] as? String ??  String(data["trans"] as? Int ?? 0)
```

Comment:
The previous implementation was terminating the flow if there was no "trans". Can you please double-check if it's ok to pass "0" as default if we don't get this param?

Reply:
For context,  I noticed that depending on the payment channel the backend returns `trans` (the transaction ID) as either a string or an integer. The transaction id is set to 0 if no transaction id is set. Ideally there should always be a transaction id returned.